### ISSUE TITLE
Unit tests fail

### ISSUE THREAD

**Issue by brandonroth** [ckrey/MQTT-Client-Framework#90]
Unit tests fail with the error

```duplicate symbol _OBJC_CLASS_$_MQTTTransport in:
    /Users/brandonroth/Library/Developer/Xcode/DerivedData/MQTTClient-anaiwgdkbhmptceidccbfugubwif/Build/Intermediates/MQTTClient.build/Debug/MQTTClientOSXTests.build/Objects-normal/x86_64/MQTTTransport.o
    /Users/brandonroth/Library/Developer/Xcode/DerivedData/MQTTClient-anaiwgdkbhmptceidccbfugubwif/Build/Products/Debug/libPods-MQTTClientOSXTests-MQTTClient.a(MQTTTransport.o)
duplicate symbol _OBJC_METACLASS_$_MQTTTransport in:
    /Users/brandonroth/Library/Developer/Xcode/DerivedData/MQTTClient-anaiwgdkbhmptceidccbfugubwif/Build/Intermediates/MQTTClient.build/Debug/MQTTClientOSXTests.build/Objects-normal/x86_64/MQTTTransport.o
    /Users/brandonroth/Library/Developer/Xcode/DerivedData/MQTTClient-anaiwgdkbhmptceidccbfugubwif/Build/Products/Debug/libPods-MQTTClientOSXTests-MQTTClient.a(MQTTTransport.o)
duplicate symbol _OBJC_IVAR_$_MQTTTransport.runLoop in:
    /Users/brandonroth/Library/Developer/Xcode/DerivedData/MQTTClient-anaiwgdkbhmptceidccbfugubwif/Build/Intermediates/MQTTClient.build/Debug/MQTTClientOSXTests.build/Objects-normal/x86_64/MQTTTransport.o
    /Users/brandonroth/Library/Developer/Xcode/DerivedData/MQTTClient-anaiwgdkbhmptceidccbfugubwif/Build/Products/Debug/libPods-MQTTClientOSXTests-MQTTClient.a(MQTTTransport.o)
duplicate symbol _OBJC_IVAR_$_MQTTTransport.state in:
    /Users/brandonroth/Library/Developer/Xcode/DerivedData/MQTTClient-anaiwgdkbhmptceidccbfugubwif/Build/Intermediates/MQTTClient.build/Debug/MQTTClientOSXTests.build/Objects-normal/x86_64/MQTTTransport.o
    /Users/brandonroth/Library/Developer/Xcode/DerivedData/MQTTClient-anaiwgdkbhmptceidccbfugubwif/Build/Products/Debug/libPods-MQTTClientOSXTests-MQTTClient.a(MQTTTransport.o)
duplicate symbol _OBJC_IVAR_$_MQTTTransport.delegate in:
    /Users/brandonroth/Library/Developer/Xcode/DerivedData/MQTTClient-anaiwgdkbhmptceidccbfugubwif/Build/Intermediates/MQTTClient.build/Debug/MQTTClientOSXTests.build/Objects-normal/x86_64/MQTTTransport.o
    /Users/brandonroth/Library/Developer/Xcode/DerivedData/MQTTClient-anaiwgdkbhmptceidccbfugubwif/Build/Products/Debug/libPods-MQTTClientOSXTests-MQTTClient.a(MQTTTransport.o)
duplicate symbol _OBJC_IVAR_$_MQTTTransport.runLoopMode in:
    /Users/brandonroth/Library/Developer/Xcode/DerivedData/MQTTClient-anaiwgdkbhmptceidccbfugubwif/Build/Intermediates/MQTTClient.build/Debug/MQTTClientOSXTests.build/Objects-normal/x86_64/MQTTTransport.o
    /Users/brandonroth/Library/Developer/Xcode/DerivedData/MQTTClient-anaiwgdkbhmptceidccbfugubwif/Build/Products/Debug/libPods-MQTTClientOSXTests-MQTTClient.a(MQTTTransport.o)
ld: 6 duplicate symbols for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)```

Using Xcode Version 7.2 (7C68) on commit 68f4f515c3783b1a7feaccd4957866d0f9483b0a

**Comment by ckrey** [comment]
would you try to run
- in the shell: pod update
- in Xcode: Product/Clean, Product/Build
and check if you included the MQTTClient files in addition to to cocoapods.

Sometimes rm -rf /Users/brandonroth/Library/Developer/Xcode/DerivedData/ helps

**Comment by brandonroth** [comment]
I've tried that.  I'm able to build the MQTTFramework target but get the linker error on the MQTTClientTests.  To recreate the issue on my machine I:

1. Cloned the repo into ~/
2. opened ~/MQTT-Client-Framework/MQTTClient in the shell
3. ran pod install
4. opened the xcworkspace and tried to run the tests.
5. tried to troubleshoot by cleaning, removing derived data, and trying again.

I also tried, just in case, restarting Xcode and finally restarting my computer.
